app/routes/overview/components/CompactEvents.css

```diff
@@ -10,7 +10,7 @@
 
 .compactEvents h3 {
   margin-bottom: 10px;
-  padding: 0;
+  padding: 0 10px;
```

Comment:
I'm not so sure about this. I feel like the old style is better. The indented title looks kinda "off" to me :thinking:

Reply:
I was going back and forth for like 15min, haha. Both look kinda "off", but I feel this looks "less" off somehow. But I'm open for a vote

Reply:
I agree with @LudvigHz on that it looks a bit off, but perhaps we should be consistent on the  headers?

Reply:
I don't really care if they're indented or not, but they should for sure be the same as the rest of the headers. So if we go back to the old style we should change all the other header to fit that styling too

Reply:
I agree that it looks a little off, but I think it looks better than it being inconsistent